src/kokkos/ekat_math_utils.hpp

```diff
@@ -82,34 +74,18 @@ Real rel_diff (const Real& a, const Real& b) {
 
 } // namespace impl
 
-template<typename ScalarT>
```

Comment:
All these functions have been removed, in favor of using Kokkos' interfaces. On the "invalid" side, I don't think it should be up to EKAT to decide what constitutes an "invalid" number. Before, ScalarTraits was implementing invalid=NaN for FP types, and invalid=very_large_int for integer types. I find this inappropriate. Let the apps decide what invalid is for them.

Reply:
Yes. We use it to initialize eamxx fields with "invalid" values, hoping to catch any usage of uninitialized stuff.